package.json

```diff
@@ -69,16 +74,18 @@
     "karma-webpack": "^2.0.4",
     "mocha": "^5.0.4",
     "moxios": "^0.4.0",
+    "node-sass": "^4.8.3",
     "nyc": "^11.6.0",
     "on-build-webpack": "^0.1.0",
     "raw-loader": "^0.5.1",
     "react-dom": "^16.2.0",
     "redux-logger": "^3.0.6",
     "redux-mock-store": "^1.5.1",
     "rimraf": "^2.5.2",
-    "style-loader": "^0.13.1",
+    "sass-loader": "^6.0.7",
+    "style-loader": "^0.13.2",
     "url-loader": "^0.5.7",
-    "webpack": "^1.12.13",
+    "webpack": "^1.15.0",
```

Comment:
This is a legacy and perhaps unsupported version of webpack. Is there any reason for using this? Why not version 3 or 4(the latest)

Reply:
Thank you @segunolalive for the observation. The version of the webpack is from the` owa-generator`. Migrating to webpack 3 or 4 is a good candidate for another task. Let me discuss this with my teammates. @dkayiwa @zeze111  @geofrocker @fuchodeveloper  @larrystone @Betty-Kebenei